test/webclient/cypress/integration/roller_spec.js

```diff
@@ -98,48 +285,85 @@ describe('AIR Receipt UI Tests', function () {
 
     });
 
+    // -- Change business to REX --
+    it('Change business to REX', function () {
+
+        // get business id from appSettings variable for 'REX'
+        appSettings.BizMap.forEach(function (item) {
+            if (item.BUD === constants.testBiz) {
+                constants.testBizID = item.BID;
+            }
+        });
+
+        // Now change the business to REX
+        cy.get('[name="BusinessSelect"]').select(constants.testBiz);
+
+        // Check BusinessSelect value is set per the expected BID from appSettings variable
+        cy.get('[name="BusinessSelect"]').should('have.value', constants.testBizID.toString());
+
+        // onSuccessful test set BID value. If above test get fail below code will not be executed.
+        constants.BID = constants.testBizID;
+
+    });
+
 
     /*****************************************************
      * Tests for node
-     * 1. Route the receipt API
-     * 2. Check http status
-     * 3. Check key `status` in API endpoint responseBody
      *******************************************************/
     it('Left side node', function () {
 
-        // Starting a server to begin routing responses to cy.route()
-        cy.server();
-
-        // To manage the behavior of network requests. Routing the response for the requests.
-        cy.route(testConfig.methodType, getAPIEndPoint(testConfig.sidebarID)).as('getReceipts');
-
         // It should be visible and selected
         cy.get(selectors.getNodeSelector(testConfig.sidebarID))
             .scrollIntoView()
             .should('be.visible')
             .should('have.class', 'w2ui-selected')
             .click().wait(constants.WAIT_TIME);
 
+
+    });
+
+    /******************************
+     * Change date in toolbar
+     * 1. Route the receipt API
+     * 2. Check http status
+     * 3. Check key `status` in API endpoint responseBody
+     *****************************/
+    it('Change date', function () {
+
+        // Starting a server to begin routing responses to cy.route()
+        cy.server();
+
+        // To manage the behavior of network requests. Routing the response for the requests.
+        cy.route(testConfig.methodType, getAPIEndPoint(testConfig.sidebarID)).as('getRecords');
+
+        // Select date from W2UI calender
+        cy.get('[name="receiptsD1"]').click().wait(constants.WAIT_TIME);
+        cy.get('[class="w2ui-calendar-title title"]').click();
+        cy.get('[class="w2ui-jump-month"][name=' + constants.month +']').click();
+        cy.get('[class="w2ui-jump-year"][name=' + constants.year + ']').click();
+        cy.get('[date="8/1/2017"]').click();
```

Comment:
Can you please take this hard-coded value into a variable? So that it can be easily configured within that variable!

Reply:
Yes. I did give try to concatenate the string. Somehow it doesn't work. Please check line no. 345.